polaris.shopify.com/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.18.2",
   "private": true,
   "scripts": {
-    "build": "yarn gen-assets && next build",
+    "build": "yarn gen-assets && yarn playroom:build && next build",
```

Comment:
I feel like `gen-assets` could include `playroom:build`. Maybe the file should be renamed to `prebuild` or something that gives clarity to what it does. Maybe not though..

Reply:
Mixed feelings about this too, but will pull this out as a thing for us to talk about in a separate PR post merge 👍